documentation/asciidoc/accessories/iqaudio/digiamp_plus.adoc

```diff
@@ -1,22 +1,22 @@
 === Raspberry Pi DigiAMP{plus}
 
-With Raspberry Pi DigiAMP{plus}, you can connect passive stereo speakers up to 2x35W with variable output, making it ideal for use for Raspberry Pi based Hi-Fi systems.
+With Raspberry Pi DigiAMP{plus}, you can connect passive stereo speakers up to 2x35W with variable output, making it ideal for use in Raspberry Pi-based hi-fi systems.
 
-DigiAMP{plus} uses the Texas Instruments TAS5756M PowerDAC and must be powered from an external supply. It requires a 12-24V DC power source (XP Power VEC65US19 power supply is recommended).
+DigiAMP{plus} uses the Texas Instruments TAS5756M PowerDAC and must be powered from an external supply. It requires a 12-24V DC power source (the XP Power VEC65US19 power supply is recommended).
 
 image::images/DigiAMP+_Board_Diagram.jpg[width="80%"]
 
 DigiAMP{plus}’s power in barrel connector is 5.5mm x 2.5mm. 
```

Comment:
I _think_ this is probably the internal and external diameters of the barrel connector, so describing it as "is 5.5mm x 2.5mm." probably isn't right?

Reply:
I think it's length of the plug, and diameter of the barrel? Umm. This needs clarifying I guess. @tobybroberts do you have actual hardware here to look at and measure things?